lte/gateway/deploy/roles/trfserver/files/traffic_server.py

```diff
@@ -671,10 +674,12 @@ def run(self):
         )
         results = ()
         threads = ()
+        # counter for started iperf instances (in call-by-reference wrapper)
+        started_iperf_instances = [0]
```

Comment:
Maybe I don't get it, but why does it need to be an array? You only seem to operate on the first index, wouldn't an integer also do the job?

Reply:
To have a reference to the same number in each thread, you need to pass an object containing the integer. This can be done by using an array with a single element, or by implementing an object with an int field.

If you pass the integer directly, changes to the number wouldn't be transmitted outside the function.

Reply:
resolve - outdated because of refactoring